src/components/modules/puzzles/puzzlesBanner/puzzlesStyleUtils.ts

```diff
@@ -0,0 +1,5 @@
+import { neutral } from '@guardian/src-foundations/palette';
+
+export const squareBorder = `2px solid ${neutral[0]}`;
+
+export const squareBoxShadow = '0px 6px 0px rgba(0, 0, 0, 0.25)';
```

Comment:
I like this approach of pulling these into a separate file 👍

Reply:
👍🏻